composer.json

```
@@ -11,7 +11,7 @@
         "sort-packages": true
     },
     "require": {
-        "php": "~7.3.0||~7.4.0",
+        "php": "~7.3.0||~8",
```

I believe is should be set to `"php": "~7.4.0||~8.0.0"` however this should be changed in scope of https://github.com/magento/magento2/issues/31081

---

In case you are not planning to complete all scope of #31081 in the pull request, please do not change the PHP versions

---

If we do not change the dependency - we can't update composer packages.

---

For now I suggest changing to this:
```suggestion
        "php": "^7.3.0||~8.0.0",
```

---

@ihor-sviziev we can't update composer packages until all dependencies are updated. I think it makes sense to update the version together with (or after) the codebase

I'd stick to "~7.3.0||~7.4.0||~8.0.0" just in case 7.5 appears unexpectedly :D